### DISCUSSION TITLE
dcm4chee-arc-light 5.23.3 deploy error

### DISCUSSION THREAD

**Discussion by CR1753343566** [dcm4che/dcm4chee-arc-light#3296] [Q&A] [answered]
****
When I deploy the dcm4chee-arc-light 5.23.3 and start, I find a error,I think this error shouldn't happen,Please help me,Thanks @gunterze 

**To Reproduce**

1. Use command storescu storescu -c DCM4CHEE@127.0.0.1:11112 x.dicom
2. In chrome,I send a resuful request

**Expected behavior**
`WARN  [org.dcm4chee.arc.store.impl.StoreServiceImpl] (EE-ManagedExecutorService-default-Thread-41) DCM4CHEE<-STORESCU(1): Failed to store received object:
: org.jboss.weld.exceptions.UnsatisfiedResolutionException: WELD-001334: Unsatisfied dependencies for type StorageProvider with qualifiers @Default @Named WELD-001475: The following beans match by type, but none have matching qualifiers:
  - Managed Bean [class org.dcm4chee.arc.storage.filesystem.FileSystemStorageProvider] with qualifiers [@Default @Named @Any],
  - Managed Bean [class org.dcm4chee.arc.storage.cloud.CloudStorageProvider] with qualifiers [@Default @Any @Named],
  - Managed Bean [class org.dcm4chee.arc.storage.emc.ecs.EMCECSStorageProvider] with qualifiers [@Default @Any @Named]
`
![error](https://user-images.githubusercontent.com/38393031/127449346-3d65ff0d-f47d-4074-b8de-5bf14ccd7d9a.PNG)

**Screenshots**

**Desktop:**
 - OS: Windows 10
 - Browser chrome

**Comment by gunterze** [comment]
Verify if the scheme of the configured

Name | Type | Description (LDAP Attribute)
-- | -- | --
Storage URI | string | RFC2079: Uniform Resource Identifier (dcmURI)

of your [Storage Descriptor](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/storage.html) is `file:` or `jclouds:`.

**Comment by CR1753343566** [comment, marked as answer]
> 验证配置的方案是否
> 
> 名字	类型	描述（LDAP 属性）
> 存储尿	字符串	RFC2079：统一资源标识符（dcmURI）
> 您的[存储描述符](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/storage.html)是或。`file:``jclouds:`

I found I make a mistake in ldap configuration ,Thanks again
